configs.go

```diff
@@ -156,18 +151,20 @@ func (s *configsService) Set(ctx context.Context, opts SetOpts) (*Config, error)
 }
 
 // Returns configs for latest release or the latest configs if there are no releases.
-func (s *configsService) Config(app *App) (*Config, error) {
-	r, err := s.store.ReleasesFind(ReleasesQuery{App: app})
+func (s *configsService) Config(db *gorm.DB, app *App) (*Config, error) {
+	r, err := releasesFind(db, ReleasesQuery{App: app})
 	if err != nil {
 		if err == gorm.RecordNotFound {
 			// It's possible to have config without releases, this handles that.
-			c, err := s.store.ConfigsFirst(ConfigsQuery{App: app})
+			c, err := configsFind(db, ConfigsQuery{App: app})
 			if err != nil {
 				if err == gorm.RecordNotFound {
-					return s.store.ConfigsCreate(&Config{
-						App:  app,
-						Vars: make(Vars),
-					})
+					// Return an empty config.
```

Comment:
This actually exposed a problem with using `created_at` to determine what is the current config for an app. There was a case where, when setting config on an app that didn't have any releases, 2 configs would get created at about the same time; 1 empty config and 1 with the update vars. Returning an empty config here fixes that race, but using an auto incremented id to order would probably be better.